Keas.Mvc/ClientApp/components/Keys/KeyTable.tsx

```diff
@@ -81,13 +81,61 @@ export default class KeyTable extends React.Component<IProps, IState> {
                                 .includes(filter.value.toLowerCase())
                     },
                     {
-                        Header: "",
-                        headerClassName: "table-actions",
-                        filterable: false,
-                        sortable: false,
-                        resizable: false,
-                        className: "table-actions",
-                        Cell: this.renderAvailableColumn
```

Comment:
You've both fixed the render method below, and replaced with an inline method. Pick one.